python/cuml/cuml/naive_bayes/naive_bayes.py

```diff
@@ -402,6 +436,9 @@ def _partial_fit(
             check_rows=X.shape[0],
             check_dtype=expected_y_dtype,
         ).array
+        if sample_weight is not None:
+            # XXX How to use `input_to_cupy_array` here with `list` as input?
+            sample_weight = cp.asarray(sample_weight, dtype=np.float32)
```

Comment:
We don't currently support lists as input in `cuml` proper (just in `cuml.accel`). Will fix this in a followup